src/resources/views/crud/fields/relationship.blade.php

```diff
@@ -34,6 +34,18 @@
             $field['type'] = 'relationship_select';
```

Comment:
This and the 3 lines above could be condensed to
```
$field['type'] = $field['ajax'] ? 'fetch' : 'relationship_select';
```

Reply:
👍